roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
@ingvagabund in that case, I would suggest that the install workflow use `state: present` and having the upgrade workflow invoke tasks that use `state: latest`.

Otherwise, if a user re-runs the install playbook (say they are using the install playbook in Ansible tower for ongoing configuration management) they could end up with newer version of the excluder inadvertently.

Granted, it isn't a huge issue, since we would expect them to pull in a new excluder with system updates, but I'd leave that to the user's discretion.